.goreleaser.yml

```diff
@@ -7,7 +7,7 @@ archives:
     name_template: "{{ .Binary }}-{{ .Tag }}-{{ .Os }}-{{ .Arch }}"
 before:
   hooks:
-    - make tfgen
+    - make only_tfgen
```

Comment:
I was referring to https://github.com/equinix/pulumi-equinix/blob/2146471d8b95653936b0d5f3399b01e32dfa7f54/.github/workflows/release.yml#L60-L61

So it looks like we run `make only_tfgen`, create the GH release, and then run `make tfgen` again via goreleaser?  In that case I think we could remove this hook entirely.

Reply:
yep I think you're right